examples/vanilla/src/tokens.ts

```diff
@@ -57,4 +57,17 @@ export const TOKENS: Record<SupportedChainId, Token[]> = {
     new Token(SupportedChainId.BASE, '0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913', 6, 'USDC', 'USD Coin'),
     new Token(SupportedChainId.BASE, '0x60a3e35cc302bfa44cb288bc5a4f316fdb1adb42', 6, 'EURC', 'EURC'),
   ],
+  [SupportedChainId.AVALANCHE]: [
+    new Token(SupportedChainId.POLYGON, ETH_ADDRESS, 18, 'AVAX', 'Avalanche'),
+    new Token(SupportedChainId.POLYGON, '0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7', 6, 'USDT', 'Tether USD'),
+    new Token(SupportedChainId.POLYGON, '0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7', 18, 'WAVAX', 'Wrapped AVAX'),
+    new Token(SupportedChainId.POLYGON, '0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e', 6, 'USDC', 'USD Coin'),
+  ],
+  [SupportedChainId.POLYGON]: [
+    new Token(SupportedChainId.AVALANCHE, ETH_ADDRESS, 18, 'POL', 'POL  ex MATIC'),
```

Comment:
Nitpick: double spaces

```suggestion
    new Token(SupportedChainId.AVALANCHE, ETH_ADDRESS, 18, 'POL', 'POL ex MATIC'),
```